### ISSUE TITLE
RTL- Disposition of menu- Stream: Bad display of menu

### ISSUE THREAD

**Issue by Jihed525** [Meeds-io/meeds#3558]
Environment: Tribe
Step of reproduction:
- Access to the stream page
- Access to any post
- Click on the three points button on the top left of the page
Current behavior:
The menu appears with bad display: spacing and margin (see attachment)
Expected behavior:
On RTL mode, The menu option should appear with the right display

builders:9547

**Comment by boubaker** [comment]
> see attachment

Missing attachment

**Comment by Jihed525** [comment]
> > see attachment
> 
> Missing attachment

@mbensalem attachement wasn't present on task, can you add it ?

**Comment by Jihed525** [comment]
attachement

<img width="181" height="178" alt="Image" src="https://github.com/user-attachments/assets/5b15c075-173d-47a6-b7ef-464e73883d53" />

**Comment by boubaker** [comment]
Thanks
